hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
if you don't need this file... remove it...
in general, delete all the files that were generated and that you are not really using.

Reply:
Actually it breaks the build, I can probably delete it and fix the build but I think it will be too much effort. WDYT?

Reply:
yes, I agree :) it's ok